circle.yml

```diff
@@ -35,7 +35,8 @@ test:
 # appropriately tag and push the container to dockerhub
 deployment:
   hub_latest:
-    branch: "master"
+    # push all branches, they are used by fxa-dev
+    branch: /.*/
```

Comment:
Could perhaps adopt a convention and do `/^(master|feature.*|pushit.*)$/` if we don't want too many simple, or ad-hoc wip branches to be pushed into docker hub.

Otherwise, LGTM